src/bin/ion/commands/inspect.rs

```diff
@@ -456,12 +455,25 @@ impl<'a, 'b> IonInspector<'a, 'b> {
     }
 
     fn inspect_eexp(&mut self, depth: usize, eexp: EExpression<AnyEncoding>) -> Result<()> {
+        let LazyRawAnyEExpressionKind::Binary_1_1(raw_eexp) = eexp.raw_invocation().kind() else {
+            unreachable!("text e-expression")
+        };
+
         let mut formatter = BytesFormatter::new(
             BYTES_PER_ROW,
             vec![
-                // TODO: Add methods to EExpression that allow nuanced access to its encoded spans
-                // TODO: Length-prefixed and multibyte e-expression addresses
-                IonBytes::new(BytesKind::MacroId, &eexp.span().bytes()[0..1]),
+                IonBytes::new(
+                    BytesKind::MacroOpcodeAndAddress,
+                    raw_eexp.opcode_and_address_span().bytes(),
+                ),
+                IonBytes::new(
+                    BytesKind::TrailingLength,
+                    raw_eexp.length_prefix_span().bytes(),
+                ),
+                IonBytes::new(
+                    BytesKind::ArgumentEncodingBitmap,
+                    raw_eexp.bitmap_span().bytes(),
+                ),
```

Comment:
🪧 More sophisticated support for e-expressions--the address, length, and bitmap now have their own colors.